### ISSUE TITLE
Pages not at 100% zoom have misaligned textarea borders

### ISSUE THREAD

**Issue by moog16** [material-components/material-components-web#2278]
### What MDC Web Version are you using?

0.31.0

### What browser(s) is this bug affecting?

Chrome 63.0.3239.132

### What OS are you using?

Mac

### What are the steps to reproduce the bug?

1. go to [demo](https://material-components-web.appspot.com/text-field.html)
1. zoom to 110% or 125%
1. scroll to textarea
1. click into textarea

### What is the expected behavior?

Borders should be aligned to look like one border

### What is the actual behavior?

There is a gap between the border of the div and textarea

### Any other information you believe would be useful?

<img width="282" alt="screen shot 2018-02-20 at 14 37 07" src="https://user-images.githubusercontent.com/579873/36453284-6c531b9c-164c-11e8-8e37-b1f242835f40.png">

**Comment by moog16** [comment]
Going to check with Chrome team on how I should handle issues like this.

**Comment by kfranqueiro** [comment]
FWIW I haven't managed to repro this in FF or Safari on Mac, nor Edge on Windows, so it may really be Chrome-specific after all.

**Comment by moog16** [comment]
Yes you're right. I haven't checked on IE or Edge, but on FF/Safari it works just fine. I've filed a Chromium bug in the meantime: https://bugs.chromium.org/p/chromium/issues/detail?id=815029
